x-pack/plugins/ml/public/explorer/explorer_charts/explorer_chart_distribution.js

```diff
@@ -414,31 +417,69 @@ export class ExplorerChartDistribution extends React.Component {
       if (_.has(marker, 'anomalyScore')) {
         const score = parseInt(marker.anomalyScore);
         const displayScore = (score > 0 ? score : '< 1');
-        contents += `anomaly score: ${displayScore}`;
+        contents += intl.formatMessage({
+          id: 'xpack.ml.explorer.distributionChart.anomalyScoreLabel',
+          defaultMessage: 'anomaly score: {displayScore}'
+        }, { displayScore });
         if (chartType !== CHART_TYPE.EVENT_DISTRIBUTION) {
-          contents += (`<br/>value: ${formatValue(marker.value, config.functionDescription, fieldFormat)}`);
+          contents += intl.formatMessage({
+            id: 'xpack.ml.explorer.distributionChart.valueLabel',
+            defaultMessage: '{br}value: {value}'
+          }, {
+            br: '<br />',
+            value: formatValue(marker.value, config.functionDescription, fieldFormat)
+          });
           if (typeof marker.numberOfCauses === 'undefined' || marker.numberOfCauses === 1) {
-            contents += (`<br/>typical: ${formatValue(marker.typical, config.functionDescription, fieldFormat)}`);
+            contents += intl.formatMessage({
+              id: 'xpack.ml.explorer.distributionChart.typicalLabel',
+              defaultMessage: '{br}typical: {typicalValue}'
+            }, {
+              br: '<br />',
+              typicalValue: formatValue(marker.typical, config.functionDescription, fieldFormat)
+            });
           }
           if (typeof marker.byFieldName !== 'undefined' && _.has(marker, 'numberOfCauses')) {
             const numberOfCauses = marker.numberOfCauses;
             const byFieldName = mlEscape(marker.byFieldName);
             if (numberOfCauses === 1) {
-              contents += `<br/> 1 unusual ${byFieldName} value`;
-            } else if (numberOfCauses < 10) {
-              contents += `<br/> ${numberOfCauses} unusual ${byFieldName} values`;
+              contents += intl.formatMessage({
```

Comment:
__question:__ is there any reason why we can't use `plural` here?
e.g `{br} { numberOfCauses, plural, one {# unusual {byFieldName} value} other {#{plusSign} unusual {byFieldName} values}}`